release-20.2: cmd/roachtest: fix nodesource installation for sequelize/typeorm/nodejs_postgres

ZhouXing19 opened the pull request:
Backport 1/1 commits from #71039.

/cc @cockroachdb/release

---

The certificate for deb.nodesource in the installation of nodesource
seems to be expired. This commit provides a work-around per
https://github.com/nodesource/distributions/issues/1266#issuecomment-931467582.

An official fix-up should be added once
https://github.com/nodesource/distributions/issues/1266
is finished.

Release note: None
Release justification: None

Resolves #70984, #70981, #70982, #70980

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/71152)
<!-- Reviewable:end -->

ZhouXing19 committed:
Backport 1/1 commits from #71039.

/cc @cockroachdb/release

The certificate for deb.nodesource in the installation of nodesource
seems to be expired. This commit provides a work-around per
nodesource/distributions#1266 (comment).

An official fix-up should be added once
nodesource/distributions#1266
is finished.

Release note: None
Release justification: None

Resolves #70984, #70981, #70982, #70980
